File: MODUL 2 PEMROGRAMAN 1/PRAK205-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
#include <math.h>
int main (){
   int a, t, simi;
   scanf("%d", &t);
   scanf("%d", &simi);
   a = sqrt(pow(simi,2) - pow(t,2));

   printf("Alas= %d cm\n", a);
   printf("Tinggi= %d cm\n",t);
   printf("Keliling= %d cm\n", a + t + simi);
   printf("Luas= %d ^cm2", a * t / 2);
   return 0;
    
}

File: MODUL 2 PEMROGRAMAN 1/PRAK205-2310817120002-SITI RATNA DWINTA SARI.py
import math
#Input 1
t, simi = map(float, input().split()) 
a = round (math.sqrt(simi**2 - t**2))

#Output 1
print("Alas= ",a, "cm")
print("Tinggi= ",round(t), "cm")
print("Keliling= ",round (a + t + simi), "cm")
print("Luas= ", round (a * t / 2), "cm^2")

#Input 2
t1 = float (input())
simi1 = float (input())
a1 = round (math.sqrt(simi1**2 - t1**2))

#Output
print("Alas= ",a1, "cm")
print("Tinggi= ",round(t1), "cm")
print("Keliling= ",round (a1 + t1 + simi1), "cm")
print("Luas= ",round (a1 * t1 / 2), "cm^2")

